axum-extra/src/extract/host.rs

```diff
@@ -139,7 +139,7 @@ mod tests {
         assert_eq!(value, "192.0.2.60");
 
         // is case insensitive
-        let headers = header_map(&[(FORWARDED, "host=192.0.2.60;proto=http;by=203.0.113.43")]);
+        let headers = header_map(&[(FORWARDED, "HOST=192.0.2.60;proto=http;by=203.0.113.43")]);
```

Comment:
This change does not seem part of this PR right?

Reply:
Correct, it's more of a bonus, as before the case-insensitivity was not being checked in the Host extractor.
I can still remove it, if you like.

Reply:
That's nice! Maybe you could extract this as a separated PR?

Reply:
It could be done as a follow-up PR, yes

Reply:
```suggestion
        let headers = header_map(&[(FORWARDED, "host=192.0.2.60;proto=http;by=203.0.113.43")]);
```